api/services/get-orgId.ts

```diff
@@ -0,0 +1,10 @@
+import config from '../lib/config';
```

Comment:
Why  not call this file rdProfessional,ts. 

Then we have one file for all rdProfessional calls in which all functions can sit.